config/import-control.xml

```diff
@@ -58,12 +58,13 @@
   <file name="PropertyCacheFile">
     <allow class="java.math.BigInteger"/>
   </file>
+  <file name="ParserUtil">
+    <allow class="antlr.Token"/>
+    <!-- commented out until https://github.com/checkstyle/checkstyle/issues/9271 -->
+    <!-- <allow class="com.puppycrawl.tools.checkstyle.DetailAstImpl"/> -->
```

Comment:
Added this here so that we do not forget in our work on #9271 to activate this.

Reply:
Please remove.
We do not need code in advance, you can place comment to issue

Reply:
Done.